swift/ios-shoppe-demo/Model/ProductViewCell.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ProductViewCell.swift
+//  ios-shoppe-demo
+//
+//  Created by Harold Davis Jr. on 6/12/20.
+//  Copyright © 2020 Harold Davis Jr. All rights reserved.
```

Comment:
Copyright needs updating

Reply:
I will fix this and just fyi  the custom header will be resolved with this PR: https://github.com/fullstorydev/ios-shoppe-demo/pull/44